specs/latest/2.0/index.html

```diff
@@ -3117,6 +3117,18 @@ <h2 class="no-toc">Table of contents</h2>
         implementations.
     </div>
 
+    <h3>Framebuffer contents after invalidation</h3>
+    <p>
+        In OpenGL ES 3.0, after calling <code>invalidateFramebuffer</code> or <code>invalidateSubFramebuffer</code>,
+        the affected region's contents are undefined. In WebGL 2.0, it is required for the contents to either stay
```

Comment:
"affected region's contents ..." to "affected region's contents become effectively undefined."